composer.json

```diff
@@ -18,9 +18,9 @@
         }
     ],
     "require": {
-        "php": "^7.0",
+        "php": "^7.2.5",
         "yiisoft/yii2": "~2.0.4",
-        "twig/twig": "~2.7"
+        "twig/twig": "~3.0"
```

Comment:
We should release at least `2.4.x` or even `3.x` after merge.

Reply:
2.4.0, I think.